test/integration/targets/apache2_module/tasks/actualtest.yml

```diff
@@ -52,10 +64,24 @@
       - 'not enabletwo.changed'
 
 - name: disable userdir module, final run
-  apache2_module: name=userdir state=absent
+  apache2_module:
+    name: userdir
+    state: absent
   register: disablefinal
 
 - name: ensure changed on successful disable
   assert:
     that:
       - 'disablefinal.changed'
+
+- name: ensure autoindex enabled
+  apache2_module:
+    name: autoindex
+    state: present
+
+- name: force disable of autoindex  # bug #2499
+  apache2_module:
+    name: autoindex
+    state: absent
+    force: True
+  when: "ansible_os_family != 'Suse'"
```

Comment:
Is it worth repeating this task, perhaps with our face and asserting that no change was made? 

Otherwise looks good,  and thanks for tidy-up the yaml.